tests/baselines/reference/tsbuild/emptyFiles/initial-build/has-empty-files-diagnostic-when-files-is-empty-and-no-references-are-provided.js

```diff
@@ -0,0 +1,49 @@
+//// [/lib/initial-buildOutput.txt]
+/lib/tsc --b /src/with-references
+exitCode:: ExitStatus.Success
```

Comment:
this also seems to disagree with the title of the test

Reply:
Fixed